### ISSUE TITLE
crates.io on ipfs with crate management / publishing via invArch?

### ISSUE THREAD

**Issue by gilescope** [InvArch/INV4-Git#36]
Given that we now have a http interface for the crates.io index ( https://github.com/rust-lang/cargo/issues/9069 ), is it possible that we could point cargo at ipfs to get our dependencies for some / all of them? I know this is a big ask but I'm keen on unstoppability.

**Comment by jakerumbles** [comment]
Cool idea! I'm assuming that the crates.io servers are managed by the Rust Foundation at the moment?

**Comment by gilescope** [comment]
Yeah but anyone can make a mirror and tell cargo to point to that. Tbh I
would think all the rust blockchains would see this as an important step
forward to unstoppability.
